src/components/sound.js

```diff
@@ -57,16 +59,19 @@ module.exports.Component = registerComponent('sound', {
     if (data.on !== oldData.on) {
       this.updateEventListener(oldData.on);
     }
-
     // All sound values set. Load in `src`.
     if (srcChanged) {
       var self = this;
       this.audioLoader.load(data.src, function (buffer) {
         self.pool.children.forEach(function (sound) {
           sound.setBuffer(buffer);
         });
+        self.loaded = true;
```

Comment:
what if src changes? We have to set `this.loaded` to false before calling `audioLoader.load`?